src/gflownet/trainer.py

```diff
@@ -129,7 +145,7 @@ def __init__(self, hps: Dict[str, Any]):
         # Print the loss every `self.print_every` iterations
         self.print_every = self.cfg.print_every
         # These hooks allow us to compute extra quantities when sampling data
-        self.sampling_hooks: List[Callable] = []
+        self.sampling_hooks: List[Callable] = [RewardStats()]
```

Comment:
Maybe add a comment saying this computes reward percentiles by default

Reply:
Removed it, unless you think it should be there, I don't think it should be there.

Reply:
Switching to `yaml` causes the failure, I am not sure how to fix it. Does that example help? The yaml file seems very specific to recursion/valance. Should I remove it?